monkey/monkey_island/readme.md

```diff
@@ -87,6 +87,11 @@
     - `pipenv sync --dev`
     - `cd ..`
 
+    Note: If for any reason the above command does not install the dependencies
+    use:
+    - `python3.7 -m pipenv lock -r --dev > requirements.txt`
+    - `python3.7 -m pip install -r requirements.txt`
+
```

Comment:
We should probably just make the deployment scripts do this so that users don't run into this issue. We'll need to update and test the windows script as well.